### DISCUSSION TITLE
Chance CVL and DCL limit

### DISCUSSION THREAD

**Discussion by Hakuna-23** [Louisvdw/dbus-serialbattery#118] [Q&A] [answered]
Hello

Thank you for your work.
Can you please help me?
I instal the driver on my CreboGx like in the wiki and for my first time with puty it worked well.
But now i whant to chance the CVL and DCL limits and i don't understand how i can do this.
I Download the driver to my pc and change the data like in the wiki. But how can i now instal this driver on my CreboGx.
Thank you

**Comment by Louisvdw** [comment, marked as answer]
Hi 
There are 2 ways to edit it.
1. you can edit it inside the GX using SSH
2. or edit it on your PC and then only copy the utils.py over to the GX

I think that while a little different to what most people would be used to on a PC, the option to edit it on the GX is the easiest.

For 1) log into your GX using SSH  and run `nano /data/etc/dbus-serialbattery/utils.py` 
You can use the arrow keys to scroll down and edit the values you need.
User Ctrl-O to save and Ctrl-X to exit the editor, and reboot for the settings to apply.

For 2) you can edit the file in something like notepad on widows PC. Then you need a app that can do SFTP like FileZilla, connect to your GX using the same login as with SSH and copy your edited file over the existing one at /data/etc/dbus-serialbattery/utils.py
Don't copy all the files as the required file permissions will be destroyed and your driver might not start.
